style.css

```diff
@@ -494,3 +494,253 @@ input {
   color: #fff;
   z-index: 2;
 }
+
+.project-details-modal {
+  display: none;
+}
+
+.project-details-modal.show {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  background-color: rgb(96, 112, 255, 0.2);
+  backdrop-filter: blur(5px);
+  min-width: 100%;
+  min-height: 100%;
+  top: 0;
+  left: 0;
+  text-decoration: none;
+  color: black;
+  z-index: 2;
+}
+
+.modal-content {
+  background-color: #fff;
+  border-radius: 10px;
+  width: 50%;
+  height: 80%;
+  margin-bottom: 100px;
+  padding: 1rem 1rem 3rem 1rem;
+  flex-wrap: wrap;
+}
+
+.pop-up-topic {
+  font-size: 80px;
+  font-weight: 200;
+}
+
+.pop-up-list {
+  padding-bottom: 6px;
+  font-size: small;
+}
+
+.pop-up-img {
+  width: 100%;
+  height: 300px;
+}
+
+.modal-header {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+}
+
+.pop-up-details-container {
+  display: flex;
+  flex-direction: row;
+  gap: 2em;
+  justify-content: center;
+  flex-wrap: nowrap;
+  margin-top: 2rem;
+}
+
+.pop-up-text {
+  flex: 2;
+  flex-wrap: nowrap;
+  font-size: 9px;
+  gap: 2rem;
+  line-height: 14px;
+}
+
+.pop-tech {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  list-style-type: none;
+}
+
+.pop-up-tech {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  list-style-type: none;
+  color: #6070ff;
+  position: relative;
+}
+
+.pop-items {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 24px;
+  font-size: 16px;
+}
+
+.canopy-pc {
+  list-style-type: none;
+  color: black;
+  font-size: 16px;
+}
+
+.pop-up-tech-type {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin: 4px 4px 0;
+  font-size: 12px;
+}
+
+.pop-up-button {
+  display: flex;
+  width: 100%;
+  gap: 1rem;
+  margin-top: 2rem;
+}
+
+.pop-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 14px;
+  gap: 5px;
+  padding: 6px 6px;
+  color: #6070ff;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  width: max-content;
+}
+
+.pop-button-source {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 14px;
+  gap: 5px;
+  padding: 6px 6px;
+  color: #6070ff;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  width: max-content;
+}
+
+.mobile-project-modal {
+  display: none;
+}
+
+.mobile-project-modal.show {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  background-color: rgb(96, 112, 255, 0.2);
+  backdrop-filter: blur(5px);
+  min-width: 100%;
+  min-height: 100%;
+  top: 0;
+  left: 0;
+  text-decoration: none;
+  color: black;
+  z-index: 2;
+}
+
+.mobile-modal-content {
```

Comment:
- [ ] Please 🙏🏻  adhere to the project requirement because I can see you are not using the correct `font-color`. which makes your design `inconsistent`. Kindly have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-30 21-25-38](https://github.com/Kwamay/Porfolio_Setup_Project/assets/66476329/d5eb7532-cbc7-4598-82eb-7a6d45b2c0ba)

</td>
<td>

![Screenshot from 2023-06-30 23-59-52](https://github.com/Kwamay/Porfolio_Setup_Project/assets/66476329/e52b4bdc-c24f-4979-84fc-be23126f8d8f)

</td>
</tr>
</table>